ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/entity/SourceDescriptionFactory.java

```diff
@@ -34,7 +41,11 @@ public static SourceDescription create(
       final boolean extended,
       final List<RunningQuery> readQueries,
       final List<RunningQuery> writeQueries,
-      final Optional<TopicDescription> topicDescription
+      final Optional<TopicDescription> topicDescription,
+      final Optional<ConsumerGroupDescription> consumerGroupDescription,
+      final Map<TopicPartition, ListOffsetsResultInfo> topicAndStartOffsets,
+      final Map<TopicPartition, ListOffsetsResultInfo> topicAndEndOffsets,
+      final Map<TopicPartition, OffsetAndMetadata> topicAndConsumerOffsets
```

Comment:
I think it would make sense to encapsulate all of this into a single class and have one `Map<TopicPartition, SourceConsumerOffsets>`